docs/installation.md

```diff
@@ -15,7 +15,7 @@ There are three options for installing `msprime`:
 
 Pre-built binary packages for `msprime` are available through
 [conda](<https://conda.io/docs/>), and built using [conda-forge](<https://conda-forge.org/>).
-Packages for Python 3.6, 3.7 and 3.8 are available for Linux, OSX and Windows.
+Packages for Python 3.7+ are available for Linux, OSX and Windows.
```

Comment:
This section implies that conda is the only place to get the pre-built binaries, I know it is covered later on, but good to mention here.

Reply:
Good point. I've now improved the wording.